simpleline/render/screen/__init__.py

```diff
@@ -144,7 +144,7 @@ def show_all(self):
         """Prepares all elements of self.window for output and then prints them on the screen."""
         for w in self.window:
```

Comment:
This method could be on TODO list, because it is weird that we don't know what is in `self.window`.